servicetalk-grpc-internal/README.adoc

```diff
@@ -0,0 +1,20 @@
+////
+* Copyright © 2021 Apple Inc. and the ServiceTalk project authors
```

Comment:
I don't see copyright at any other `*.adoc` file we have. Do we need it here?

Reply:
Since we can, might as well start adding it. I am not sure why it would not have been included previously.